src/data/teams.json

```diff
@@ -15,10 +24,20 @@
           "slackUsername": "Mircea Teodor Oprea"
         }
       }
-    ]
+    ],
+    "responsibilities": []
   },
   {
     "title": "Back-End Developers",
+    "description": "Back-End Developers",
+    "platforms": [
+      {"name": "github", "label": "thenewboston", "link": "https://github.com/thenewboston-developers"},
```

Comment:
Should we place the specific repository links instead of the organization link?

Reply:
Yeah, we will. I wasn't sure about the data so most of it is generic. We'll fill it in later.